content/en/blog/2022-05-05-introducing-kluctl-and-flux.md

```diff
@@ -0,0 +1,83 @@
+
+---
+title: "Kluctl And Flux"
```

Comment:
title: Fresh new glue - Kluctl And Flux

https://giphy.com/gifs/elmersproducts-school-glue-elmers-6It3zvCbhPmQjkRwii
oder
https://giphy.com/gifs/heyarnold-hey-arnold-nicksplat-xT1R9MqYPoHGswOJ9K